packages/web/src/components/typography/constants.ts

```diff
@@ -16,155 +11,36 @@ export const fontWeightMap: Record<FontWeight, number> = {
   ultraLight: 100
 }
 
-export const variantInfoMap: Record<TextVariant, TextVariantSizeInfo> = {
+export const variantTagMap: Record<TextVariant, VariantSizeTagMap> = {
   display: {
-    XL: {
-      tag: 'h1',
-      fontSize: '96px',
-      lineHeight: '120px'
-    },
-    L: {
-      tag: 'h1',
-      fontSize: '72px',
-      lineHeight: '88px'
-    },
-    M: {
-      tag: 'h1',
-      fontSize: '56px',
-      lineHeight: '68px'
-    },
-    S: {
-      tag: 'h1',
-      fontSize: '42px',
-      lineHeight: '52px'
-    }
+    XLarge: 'h1',
+    Large: 'h1',
+    Medium: 'h1',
+    Small: 'h1'
   },
   heading: {
-    XL: {
-      tag: 'h1',
-      fontSize: '36px',
-      lineHeight: '40px'
-    },
-    L: {
-      tag: 'h2',
-      fontSize: '28px',
-      lineHeight: '32px'
-    },
-    M: {
-      tag: 'h3',
-      fontSize: '24px',
-      lineHeight: '32px'
-    },
-    S: {
-      tag: 'h4',
-      fontSize: '20px',
-      lineHeight: '24px'
-    }
+    XLarge: 'h1',
+    Large: 'h2',
+    Medium: 'h3',
+    Small: 'h4'
   },
   title: {
-    L: {
-      tag: 'p',
-      fontSize: '18px',
-      lineHeight: '24px'
-    },
-    M: {
-      tag: 'p',
-      fontSize: '16px',
-      lineHeight: '20px'
-    },
-    S: {
-      tag: 'p',
-      fontSize: '14px',
-      lineHeight: '16px'
-    },
-    XS: {
-      tag: 'p',
-      fontSize: '12px',
-      lineHeight: '16px'
-    }
+    Large: 'p',
+    Medium: 'p',
+    Small: 'p',
+    XSmall: 'p'
   },
   label: {
-    XL: {
-      tag: 'label',
-      fontSize: '20px',
-      lineHeight: '24px',
-      letterSpacing: '0.5px',
-      textTransform: 'uppercase'
-    },
-    L: {
-      tag: 'label',
-      fontSize: '16px',
-      lineHeight: '16px',
-      letterSpacing: '0.5px',
-      textTransform: 'uppercase'
-    },
-    M: {
-      tag: 'label',
-      fontSize: '14px',
-      lineHeight: '16px',
-      letterSpacing: '0.5px',
-      textTransform: 'uppercase'
-    },
-    S: {
-      tag: 'label',
-      fontSize: '12px',
-      lineHeight: '12px',
-      letterSpacing: '0.5px',
-      textTransform: 'uppercase'
-    },
-    XS: {
-      tag: 'label',
-      fontSize: '10px',
-      lineHeight: '12px',
-      letterSpacing: '0.5px',
-      textTransform: 'uppercase'
-    }
+    XLarge: 'label',
```

Comment:
so just a headsup, "label" element has to be associated with an "input" element, just making sure that would actually be the case for all label types. otherwise span/p or whatever could just work

Reply:
yea I assumed that's what these would be used for. can change later if we want to render something different 👍